front-end/src/pages/FollowListPage.tsx

```diff
@@ -0,0 +1,93 @@
+import { RouteComponentProps } from 'react-router-dom';
+import { Row, Col, Card, CardBody, CardTitle, Container, Button, CardText, Nav, NavItem, NavLink, TabContent, TabPane } from 'reactstrap';
+import AuthorListItem from '../components/AuthorListItem';
+import { Author } from '../types/Author';
+import { UserLogin } from '../types/UserLogin';
+
+interface Props extends RouteComponentProps<MatchParams>{
+  loggedInUser: UserLogin | undefined,
+  activeTab: string,
+}
+
+interface MatchParams {
+  authorId: string;
+}
+/**
+ * Render list of search results when searching for an author by display name
+ * @param props 
+ */
+export default function FollowListPage(props: Props) {
+  const [followers, setFollowers] = useState<Author[] | undefined>(undefined);
+  const [following, setFollowing] = useState<Author[] | undefined>(undefined);
+
+  // get all followers and following
+  useEffect(() => {
+    axios.get(process.env.REACT_APP_API_URL + "/api/author/" + props.loggedInUser?.authorId + "/followers/").then(res => {
+      console.log(res.data.items)
+      const followersList: Author[] = res.data.items;
+      setFollowers(followersList);
+    });
+
+    axios.get(process.env.REACT_APP_API_URL + "/api/author/" + props.loggedInUser?.authorId + "/following/").then(res => {
+      console.log(res.data)
+      const followingList: Author[] = res.data.items;
+      setFollowing(followingList);
+    });
+
+  }, []);
+
+  const [activeTab, setActiveTab] = useState(props.activeTab);
+
+  const toggle = (tab: string) => {
+    if (activeTab !== tab) setActiveTab(tab);
+    props.history.push("/author/" + props.loggedInUser?.authorId + "/" + tab);
+  }
+
+  return (
+    <Container fluid>
+
+      <Row className="justify-content-md-center">
+        <Nav tabs>
+          <NavItem>
+            <NavLink onClick={() => { toggle('followers') }} className="mb-2 text-muted">
+              <h2>Followers</h2>
+            </NavLink>
+          </NavItem>
+          <NavItem>
+            <NavLink onClick={() => { toggle('following') }} className="mb-2 text-muted">
+              <h2>Following</h2>
+            </NavLink>
+          </NavItem>
+        </Nav>
+      </Row>
+      <Row className="justify-content-md-center">
+        <Col sm="12" md={{ size: 6 }}>
+          <TabContent activeTab={activeTab}>
+            <TabPane tabId="followers">
+              <Row>
+                <Col sm="12">
+                  {followers?.length !== 0 ?
+                    followers?.map((follower: Author) => <AuthorListItem author={follower} loggedInUser={props.loggedInUser}></AuthorListItem>) :
+                    <Card body className="text-center"><CardBody><CardTitle tag="h5" >You have no followers :(</CardTitle></CardBody></Card>
+                  }
+                </Col>
+              </Row>
+            </TabPane>
+            <TabPane tabId="following">
+              <Row>
+                <Col sm="12">
+                  {following?.length !== 0 ?
```

Comment:
We might want to use the optional chaining operator less frequently going forward. There could potentially be bugs that pop-up if `following` becomes undefined for a given reason. 

We may want to go for something like this
```ts
if (following) {
    // do thing
    console.log(following.length)
} else {
    // warn here
    console.error('Following is undefined when it shouldn\'t be!')
}
```
As it'll notify us if our assumptions about `following` existing ever breaks